src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__item {
+  display: inline-block;
+  padding-right: 20px;
```

Comment:
```suggestion
  margin-right: 20px;
```

but 
![image](https://user-images.githubusercontent.com/64437769/142611701-a4986809-0513-4774-b9a7-a6b15b8668af.png)

![image](https://user-images.githubusercontent.com/64437769/142611661-fd4cf4e3-24ce-46ae-9627-ac84454a9556.png)